tests/test_opensearch.py

```diff
@@ -175,8 +246,14 @@ def elasticsearch_7_10_fgac_client(domain_endpoint_elasticsearch_7_10_fgac, open
     return client
 
 
+@pytest.fixture(scope="session")
+def opensearch_serverless_client(opensearch_serverless_collection_endpoint):
+    client = wr.opensearch.connect(host=opensearch_serverless_collection_endpoint)
+    return client
+
+
 # testing multiple versions
-@pytest.fixture(params=["opensearch_1_0_client", "elasticsearch_7_10_fgac_client"])
+@pytest.fixture(params=["opensearch_1_0_client", "elasticsearch_7_10_fgac_client", "opensearch_serverless_client"])
 def client(request):
     return request.getfixturevalue(request.param)
```

Comment:
This seems to be causing an issue for tests when the fixture isn't ready yet because serverless collection didn't finish creating. Looking into it...

Reply:
Is this what's raising the `BulkIndexError` during testing?

Reply:
Looks like this is something else. The reason for `BulkIndexError ` is - `{'type': 'status_exception', 'reason': 'wait_for refresh policy is not supported.'}` - we try to refresh the shard when we index the document and serverless seems to not support this specific way of doing it

Reply:
Ok looks like neither refresh policy nor refresh endpoint aren't supported in opensearch serverless yet but according to the [docs](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-overview.html) the refresh interval is between 10 and 30 seconds depending on the size of the requests. I've replaced refresh policy with sleeps in the tests at least for now until there isn't a better solution